IPython/core/display.py

```diff
@@ -167,7 +182,14 @@ def display(*objs, **kwargs):
             if metadata:
                 # kwarg-specified metadata gets precedence
                 _merge(md_dict, metadata)
-            publish_display_data(data=format_dict, metadata=md_dict)
+            publish_display_data(data=format_dict, metadata=md_dict,
+            **kwargs)
+
+
+def update_display(*objs, **kwargs):
+    """Update an existing display"""
+    kwargs['update'] = True
+    return display(*objs, **kwargs)
```

Comment:
👍 I like that I can use `display` with `update` = True and not have to use `update_display` directly as a function. :smile: /cc @ivanov